app/src/main/java/com/example/defaultstyles/CustomCircleView.kt

```diff
@@ -15,12 +15,18 @@ private const val DEFAULT_HAS_BORDER = false
 class CustomCircleView : View {
 
     constructor(context: Context) : this(context, null)
-    constructor(context: Context, attrs: AttributeSet?) : this(context, attrs, 0)
+    constructor(context: Context, attrs: AttributeSet?) : this(context, attrs, DEFAULT_STYLE_ATTR)
```

Comment:
If we don't pass this up to the `View` constructor, we don't give it the opportunity to apply any `View` properties.

Now, if we add a `View` (i.e. not custom) property to the default style, e.g. `android:padding` it'll be taken into account.

Reply:
That's why I had trouble passing `android:color` :). Thanks!